reference-architecture/vmware-ansible/roles/static-ipaddress/templates/ifcfg-public.j3

```diff
@@ -0,0 +1,13 @@
+TYPE=Ethernet
```

Comment:
It is intentional actually, the j3 template creates a subinterface that isnt disrupted when the network restart occurs. This way I dont lose connectivity with my existing ssh session and I can continue. After its activity the j2 template lays down the actual IP address. I guess I could parameterize the var in the config file and reduce the need for 2 separate files?